lib/seal.rb

```diff
@@ -41,4 +42,10 @@ def bark_at(team, mode: nil)
   rescue StandardError => e
     puts "Error barking at team '#{team.name}': #{e.message}"
   end
+  
+  def send_quotes_message(team, mode)
+    if team.quotes_days.map(&:downcase).include?(Date.today.strftime("%A").downcase) && team.public_send("#{mode}")
```

Comment:
Why the addition of the `&& team.public_send`? Everything else (except the `sleep 60`) is a straightforward refactor, but this is a change of behaviour. If it's needed, can it be pulled into its own commit?

Reply:
It's because we now call the Seal with either `morning_seal_quotes` or `afternoon_seal_quotes` and this is a short way to check if the team has enabled the alerts with the same name in their YAML config. I'll add something to the commit message. 

See https://stackoverflow.com/questions/1407451/calling-a-method-from-a-string-with-the-methods-name-in-ruby

Reply:
Thanks Murilo - I think this could be made a bit more readable (and avoid the engineer having to read the commit message to understand why the `public_send` was added), something like:

```suggestion
    today_is_quote_day = team.quotes_days.map(&:downcase).include?(Date.today.strftime("%A").downcase)
    team_has_enabled_this_mode = team.public_send("#{mode}")
    today_is_quote_day && team_has_enabled_this_mode
```